contrib/implements/main.go

```diff
@@ -62,9 +68,31 @@ func main() {
 	}
 	switch {
 	case reportJSON:
-		r = implements.NewJSONReport(o, os.Stdout)
-	default:
-		r = implements.NewTextReport(o)
+		rpts = append(rpts, implements.NewJSONReport(o, os.Stdout))
```

Comment:
Is it possible that user chooses format as json (reportJSON = True) but report file as text(outputText = '-' or 'someFile.txt') or vice versa, will it be allowed. What will happen in that case?
I guess it is allowed from the code.

Reply:
Yeah, in these changes the `--json` option is now pretty much an alias for `--report-json -` so you can combine `--json` with the text report in any way that you can with `--report-json`. That includes output both to stdout, which isn't super useful but allowed.